pkg/scheduler/framework/plugins/requestedtocapacityratio/requested_to_capacity_ratio.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
```

Comment:
Looks like spacing is wrong on these headers so it's failing boilerplate check, see https://github.com/kubernetes/kubernetes/blob/9c4f2a81798a310898d632dc8aabbcb0a8b4e33f/pkg/scheduler/framework/plugins/imagelocality/image_locality.go#L1-L15